site/components/src/main/java/uk/nhs/digital/ps/beans/Series.java

```diff
@@ -255,4 +257,13 @@ protected <T extends HippoBean> List<T> getRelatedDocuments(
         return toList(query.execute().getHippoBeans());
     }
 
+    // because there is no publicationsystem:NominalDate variable for series we need to use either
```

Comment:
This would be more visible if it were a JavaDoc so it would appear in the doc pop ups. And then you'd need to specify which field is actually being used.

Reply:
thanks @lee-richardson-nhs I will ensure this is done going forward. :)